demo/src/modules/home/containers/TableContainer.js

```diff
@@ -153,8 +168,21 @@ export default function (props) {
 	const onDownload = () => initData.data;
 	props.app.addHelpButton("https://github.com/TeskaLabs/asab-webui/blob/master/demo/src/modules/home/containers/TableContainer.js");
 
+	const onSort = (value) => setSort(value);
+
+	const buttonWithAuthz = {
+		title:"Button with authz",
+		color:"danger",
+		size:"sm",
+		onClick() {alert("You've clicked button with authz")},
+		resource: "res:res",
+		resources: ["res:res"],
+		children: (<><i className="cil-trash mr-2"></i>ButtonWithAuthz</>)
+	}
+
 	return (
 		<Container>
+			<Button onClick={() => setLoading(prev => !prev)} className="mb-2">Set Loading</Button>
```

Comment:
@titkov49 Should be this button for testing the data loading in here?

Reply:
I kept it in order to show full functionality of DataTable in demo